content/v1.0.0/index.md

```diff
@@ -0,0 +1,231 @@
+
+### How does Conventional Commits handle revert commits?
+
+Reverting code can be complicated: are you reverting multiple commits? if you revert a feature, should the next release
+instead be a patch?
+
+Conventional Commits does not make an explicit effort to define revert behavior. Instead we leave it to tooling
+authors to use the flexility of _types_ and _footers_ to develop their logic for handling reverts.
+
+One recommendation is to use the `revert` type, and a footer that references the commit SHAs that are being reverted:
+
+```
+revert: let us never again speak of the noodle incident
+
+Refs: #676104e, #a215868
+```
+
+## About
+
+The Conventional Commit specification is inspired by, and based heavily on, the [Angular Commit Guidelines](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines).
+
+The first draft of this specification has been written in collaboration with some of the folks contributing to:
+
+* [conventional-changelog](https://github.com/conventional-changelog/conventional-changelog): a set of tools for parsing conventional commit messages from git histories.
+* [bumped](https://bumped.github.io): a tool for releasing software that makes it easy to perform actions before and after releasing a new version of your software.
+* [unleash](https://github.com/netflix/unleash): a tool for automating the software release and publishing lifecycle.
+* [lerna](https://github.com/lerna/lerna): a tool for managing monorepos, which grew out of the Babel project.
+
+## Tooling for Conventional Commits
+
+* [fastlane-plugin](https://github.com/xotahal/fastlane-plugin-semantic_release): follows the specification to manage versions and generate a changelog automatically
+* [php-commitizen](https://github.com/damianopetrungaro/php-commitizen): a tool built to create commit messages following the Conventional Commit specs. 
+Configurable and usable for PHP projects as a composer dependency or usable globally for non-PHP projects.
+* [conform](https://github.com/autonomy/conform): a tool that can be used to enforce policies on git repositories, including conventional commits.
+* [standard-version](https://github.com/conventional-changelog/standard-version): Automatic versioning and CHANGELOG management, using GitHub's new squash button and the recommended Conventional Commits workflow.
+* [Git Commit Template](https://plugins.jetbrains.com/plugin/9861-git-commit-template): Add _Conventional Commits_ support to [JetBrains Editors](https://www.jetbrains.com/) (IntelliJ IDEA, PyCharm, PhpStorm...).
+* [commitsar](https://github.com/commitsar-app/commitsar): Go tool for checking if commits on branch are conventional commit compliant. Comes with Docker image for CI uses.
+* [semantic-release](https://github.com/semantic-release/semantic-release): A tool that automates the whole package release workflow including: determining the next version number, generating the release notes and publishing the package.
```

Comment:
Feel free to mention https://github.com/rustic-games/conventional-commit here as well.

Reply:
https://github.com/conventional-commits/conventionalcommits.org/blob/9678e4bc4c557a78927d0dee57f6368042604e13/index.md#L230

Reply:
https://github.com/conventional-commits/conventionalcommits.org/blob/9678e4bc4c557a78927d0dee57f6368042604e13/content/next/index.md#L230

Reply:
Yeah, will happily add it, but let's add as a follow up as soon as we land this.